etc/kayobe/ansible/nova-compute-drain.yml

```diff
@@ -5,68 +5,39 @@
   tags:
     - nova-compute-drain
   vars:
-    venv: "{{ virtualenv_path }}/openstack"
-    live_migration_fatal: true
+    nova_compute_migration_fatal: true
   tasks:
```

Comment:
Would it make sense to put this in a main.yml, using include_tasks instead?

Reply:
Good call.